documentation/release_notes.rst

```diff
@@ -8,6 +8,51 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.9.0
+===============
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``fill``, ``move``, ``replace``, ``replace_if``,
+``remove``, ``remove_if``, ``mismatch``, ``minmax_element``, ``min``, ``max``, ``find_first_of``, ``find_end``, ``is_sorted_until``
+- Improved performance of set operation algorithms (e.g., ``set_union``, ``set_difference``, etc.)
+
+Fixed Issues
+------------
+- Eliminated runtime exceptions that were encountered when compiling code that uses device policies with the open source
+oneAPI DPC++ Compiler (clang++ driver) without specifying an optimization flag.
+- Eliminated compile time warnings throughout the library.
+
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+
+Existing Issues
+^^^^^^^^^^^^^^^
+See oneDPL Guide for other `restrictions and known limitations`_.
+
+- Incorrect results may be observed when calling ``sort`` with a device policy on Intel® Arc™ graphics 140V with data
+  sizes of 4-8 million elements.
+- ``sort``, ``stable_sort``, ``sort_by_key`` and ``stable_sort_by_key`` algorithms fail to compile
+  when using Clang 17 and earlier versions, as well as compilers based on these versions,
+  such as Intel® oneAPI DPC++/C++ Compiler 2023.2.0.
+- ``histogram`` algorithm requires the output value type to be an integral type no larger than four bytes
+  when used with an FPGA policy.
+- ``histogram`` may provide incorrect results with device policies in a program built with ``-O0`` option.
```

Comment:
Does it happen with specific hardware or their types (GPU/CPU)? Is there the same issue in Windows (e.g. with /O2 option)?

Reply:
Good question.  This was a GPU specific issue which has since been fixed by an update in the drivers.  We can remove this line.

Reply:
It looks like (rolling) version 2506.18. I'll see if I can figure out the LTS version.

Reply:
Info on the release dates of the rolling and LTS drivers is available at https://dgpu-docs.intel.com/releases/releases.html

Reply:
I believe it is not specific to OS or GPU platform. This issue was showing up on windows and with integrated graphics as well as Intel Data Center GPU Max 1550. It also seems invariant of compiler version.  The driver seems to be the culprit.

Reply:
Looks good.  resolving.